alembic/versions/2025_04_30_8e0b9e03054d_add_entries_for_raredisease_in_order_.py

```diff
@@ -0,0 +1,51 @@
+"""add entries for raredisease in order type application
+
+Revision ID: 8e0b9e03054d
+Revises: 039dbdf8af01
+Create Date: 2025-04-30 09:56:53.670128
+
+"""
+
+from typing import List
```

Comment:
we use the typping from the python library, so we don't import python types like list, set, etc.
```suggestion
```